### PR TITLE
Issue #35 Adding sshfs

### PR DESCRIPTION
**Pull request by hferentschik**

### CODE REVIEW

**Comment by hferentschik** [comment]
Not sure whether the test assertion will work, since I cannot run the test locally atm. Let's wait for the CI build

**Comment by hferentschik** [comment]
@budhrg any idea why this CI build failed right now?

**Comment by hferentschik** [comment]
retest this please

**Comment by hferentschik** [comment]
retest this please

**Comment by hferentschik** [comment]
@praveenkumar or @budhrg, can one of you guys please merge once it is confirmed the test worked as well.

File: iso/Dockerfile [hunk 1]
```diff
@@ -33,11 +33,12 @@ ADD socat $ROOTFS/usr/bin
...
-# Add cifs-utils to mount fileshares
-ENV CIFS_DEPS cifs-utils \
-              samba-libs
+# Add cifs-utils and sshfs-fuse to mount fileshares
+ENV HOST_FOLDER_DEPS cifs-utils \
+              samba-libs \
+              sshfs-fuse
 
-RUN for dep in $CIFS_DEPS; do \
+RUN for dep in $HOST_FOLDER_DEPS; do \
```
Comment (brgnepal) [inline iso/Dockerfile:41]: :+1:

**Comment by brgnepal** [comment]
Waiting for this job to finish https://ci.centos.org/job/minishift-b2d-iso-pr/47/

Implementation looks good to me :+1:

**Comment by hferentschik** [comment]
In case you want to try:

* Make sure you have a ssh daemon running on your host (pretty much default on Linux and OS X)
* Start Minishift
* `$ minishift ssh`
* Create mountpoint e.g. _mkdir /home/docker/data/_
* Mount for example `sudo sshfs \<user\>@\<host ip\>:\<host path\> /home/docker/data/
* `$ sudo -s`
* `$ ls /home/docker/data/`

The aim is to add manual instructions to the docs to start with an do a automated integration (even using our own ssh daemon) in a future sprint.

**Comment by brgnepal** [comment]
Verified locally and working as expected. Merged! Thanks

**Pull request merged**